dist/Documents.d.ts

```diff
@@ -19,7 +19,7 @@ declare class Documents implements Pointer.Library, Pointer.Validator, ObjectSch
     inScope(id: string): boolean;
     hasPointer(id: string): boolean;
     getPointer(id: string): Pointer.Class;
-    get(uri: string, requestOptions?: HTTP.Request.Options): Promise<[PersistedDocument.Class, HTTP.Response.Class]>;
+    get<T extends PersistedDocument.Class>(uri: string, requestOptions?: HTTP.Request.Options): Promise<[T, HTTP.Response.Class]>;
```

Comment:
This signature doesn't let you do things like:
```
get<Project>( "" ).then( ( [ project, response ]:[ Project & PersistedDocument.Class, HTTP.Response.Class ] ) => {} );
```
In order for it to allow it, the signature should be:
```
get<T>( uri:string, requestOptions?:HTTP.Request.Options ): Promise<[T & PersistedDocument.Class, HTTP.Response.Class]>;
```